src/index.tsx

```diff
@@ -814,7 +815,7 @@ const ModalizeBase = (
 
   const renderModalize = (
     <View
-      style={[s.modalize, { elevation: modalElevation }]}
+      style={[s.modalize, { elevation: modalElevation, zIndex }]}
```

Comment:
Instead of adding a new props for each property lets add an object. Maybe we can call it `modalizeStyle`, not sure if it would be confusing with `modalStyle` though.

Reply:
it's even better

Reply:
maybe `rootStyle`